package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "eslint .",
     "test": "node test.js",
     "posttest": "npm run lint",
-    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require test/init.js --require test/*.test.js --require test/interfaces/lb-bdd.js --ui lb-bdd"
+    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require test/init.js --require test/interfaces/lb-bdd.js --ui lb-bdd"
```

Comment:
The cmd I am using is
`./node_modules/.bin/_mocha --timeout 40000 --require test/init.js --require test/interfaces/lb-bdd.js --ui lb-bdd test/*.test.js`

In this case I think they are equal...